client-app/public/firebase-messaging-sw.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-undef */
```

Comment:
really need it?

Reply:
yes, because `firebase` object is exposed by imported scripts implicitly